src/Auth0Client.ts

```diff
@@ -299,6 +305,11 @@ export default class Auth0Client {
     }
 
     this.customOptions = getCustomInitialOptions(options);
+
+    this.useRefreshTokensFallback =
```

Comment:
`this.useRefreshTokensFallback = this.options.useRefreshTokensFallback !== false;`

Reply:
Oh my :D Thanks!